cogs/music.py

```diff
@@ -367,12 +370,16 @@ async def playlist(self, ctx: commands.Context, action: str = 'show', song_numbe
         elif action == 'add':
             if song_number == 'random':
                 song_number = random.randint(0, len(_songlist) - 1)
-            _playlist.append(_songlist[int(song_number) - 1])
-            await ctx.send(boxed_string(
-                _('«‎{}»‎ added to queue.').format(
-                    _songlist[int(song_number) - 1][: -len(MUSIC_EXT)]
-                )
-            ))
+            elif song_number.startswith('http'):
+                _playlist.append(song_number)
+                await ctx.send(boxed_string(f'Url {song_number} added to queue.'))
```

Comment:
Exposing a url is a potential vulnerability.
Either sanitize input (try and extract song name with a GET, for instance)
or don't print it back at all.

Reply:
done